pkg/devspace/configure/image.go

```diff
@@ -20,15 +19,17 @@ import (
 
 const dockerHubHostname = "hub.docker.com"
 const githubContainerRegistry = "ghcr.io"
-const noRegistryImage = "devspace"
+
+// const noRegistryImage = "devspace"
```

Comment:
Is this needed?

Reply:
It is not used in this file and it's not exported as well so I just commented it. Maybe it's good if I remove it.

Reply:
@pratikjagrut yes please remove it

Reply:
done.